jemalloc-debug.spec

```diff
@@ -2,7 +2,13 @@
 ### RPM external jemalloc-debug %{jemalloc_version}
 
 %build
-./autogen.sh \
+XOPTS=""
+case %{cmsplatf} in
+  # set the huge page size to 16M on ARMv8
+  *_aarch64_*) XOPTS="--with-lg-hugepage=24" ;;
+esac
```

Comment:
please use rpm buildin macros e.g. `ifarch` instead of `cmsplatf`
```suggestion
%ifarch aarch64
# set the huge page size to 16M ( log2(16M)=24 ) on ARMv8
XOPTS="--with-lg-hugepage=24"
%endif
```

Reply:
and same in other 2 specs

Reply:
Done, thanks.  That's much cleaner, was following an old example.